values.yaml

```diff
@@ -8,7 +8,7 @@ debug: false
 server:
   image:
     repository: temporalio/server
-    tag: 0.20.0
+    tag: 0.21.1
```

Comment:
Be aware of https://github.com/temporalio/temporal/pull/277, I think you may need some changes here to plumb pod IP to `TEMPORAL_BROADACAST_ADDRESS` env var

Reply:
perfect, thank you for the heads up, will keep an eye on it / make changes as needed.